From a1508aff41a526b900f958831a1196f2655c57e9 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Thu, 6 Dec 2018 11:47:52 -0800
Subject: [PATCH] corelib: Include sys/types.h for uint32_t

This has been includes indirectly on glibc/linux systems
via inttypes.h -> stdint.h -> sys/types.h but it breaks on
musl where this indirect include chain does not exist.

Upstream-Status: Pending
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 src/corelib/global/qnumeric_p.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/corelib/global/qnumeric_p.h b/src/corelib/global/qnumeric_p.h
index 86e7997680..73bcd370c2 100644
--- a/src/corelib/global/qnumeric_p.h
+++ b/src/corelib/global/qnumeric_p.h
@@ -55,6 +55,7 @@
 #include "QtCore/private/qglobal_p.h"
 #include <cmath>
 #include <limits>
+#include <sys/types.h>
 
 #if defined(Q_CC_MSVC)
 #  include <intrin.h>
